### ISSUE TITLE
Комментарии

### ISSUE THREAD

**Issue by SilentImp** [FrontenderMagazine/a-guide-to-flexbox#1]

**Comment by zendiesel** [comment]
Спасибо за статью, очень привлекательно выглядит, хотелось узнать, на сегодняшний 2015ый год, это волшебство можно использовать в продакшене?(даже используя префиксы) или все очень еще сыро и стоит все также мучиться с floatom?

**Comment by matmuchrapna** [comment]
Это можно использовать с префиксами и с запасным решением в виде `display: table` для старых ие

**Comment by zendiesel** [comment]
Спасибо. Здесь я хотел уточнить, я нахожусь в "самом начале" освоения css и много много смотрел уроков по верстке, видел сколько мучений нужно делать с float, т.е. можно перепрыгнуть этот урок с флоатом  и использовать только flex? Хотя конечно же в будущем надо будет досконально все изучить, все тонкости.

**Comment by matmuchrapna** [comment]
флоаты тоже нужно изучить так как всё равно рано или поздно придётся разбираться в чужой вёрстке

**Comment by zendiesel** [comment]
Согласен, просто мне сегодня (выгляжу как школьник наверное), нужно создать одностраничный сайт, точнее переделать его, до меня его вообще верстали, то ли в фронт-пейдже или что то подобном, все в таблице и хтмл коде, и сайт сам по себе ужасно выглядит, хотелось сделать приятно, одному старому другу, за короткое время)) А так конечно все следует знать. Спасибо еще раз маэстро.

**Comment by najmubadr** [comment]
Благодарствую, за весьма полезный труд!

**Comment by netdogg** [comment]
Вы выставляете классы всем новым атрибутам html5 
Пишите <header class="header">Header</header>
Разве сейчас не пишут <header >Header</header>
Пробовал удалить классы, тогда ломалась верстка...не понимаю(

**Comment by adarrra** [comment]
Загадочное свойство - flex-shrink. Как я поняла отсюда https://css-tricks.com/almanac/properties/f/flex-shrink/
его работу можно посмотреть только применяя вместе с flex-bazis?
Пытаюсь сделать так: когда средний контейнер упирается в два соседних - он больше не растет. Не могу придумать как сделать без задания flex-bazis

**Comment by Adriatika** [comment]
Здравствуйте! Спасибо за статью. Пожалуйста, уточните для меня, если Вас не затруднит: пишите "Flex-direction. Применяется к: родительскому элементу flex-контейнера.", в смысле к самому   контейнеру, в котором располагаем дочерние элементы? Или свойство нужно применять к родителю, который в себе содержит flex-контейнер? Этот момент для меня малость неясен. Заранее, спасибо!

**Comment by ahtohbi4** [comment]
@Adriatika, свойство не наследуется, применять нужно к самому контейнеру.

http://jsfiddle.net/ahtohbi4/n9kcm1uf/1/

**Comment by jemali-m** [comment]
Спасибо за хорошую статью !

**Comment by omar-g** [comment]
Возможно ли с помощью flexbox сделать как на скрине? https://yadi.sk/i/X6MoyfNaqZzsv

**Comment by iamstarkov** [comment]
@omar-g вероятнее стоит спросить на StackOverflow или тостере

**Comment by Adriatika** [comment]
Думаю, да.

2016-03-29 22:26 GMT+07:00 Vladimir Starkov <notifications@github.com>:

**Comment by ahtohbi4** [comment]
@omar-g, для этого скорее подойдет (пока экспериментальный) Grid Layout:

   - https://www.youtube.com/watch?v=hAFcdRrhwQM
   - https://habrahabr.ru/company/microsoft/blog/140715/

Совсем скоро во всех браузерах страны.

**Comment by Apollo108** [comment]
"Обратите внимание, она необязательно должна быть горизонтальной, всё зависит от свойства justify-content (см. ниже)."
В оригинале - flex-direction.
